README.md

```diff
@@ -1,21 +1,30 @@
-# dystonse-gtfs-importer
+# dystonse-gtfs-tool
```

Comment:
"Tool" is a very nondescript name... the name should be indicating what this tool is for. It has more than one use case obviously, but what do all the uses have in common that makes it necessary to keep them together in one crate? Maybe they all have to access the database directly (and provide interfaces for any other part of our software that might want to use data from the database)? Then we could call it `dystonse-gtfs-db` or `dystonse-gtfs-database` or maybe even `dystonse-gtfs-data`...